appveyor.yml

```diff
@@ -1,87 +1,88 @@
-#     support for.
-#   - Use CMake to ...
-#       build Cap'n Proto with MinGW.
-#       build Cap'n Proto with VS2015 and VS2017.
-#       build Cap'n Proto samples with VS2015 and VS2017.
-
-version: "{build}"
-
-branches:
-  only:
-    - master
-    - /release-.*/
-# Don't build non-master branches (unless they open a pull request).
-
-image: Visual Studio 2017
-# AppVeyor build worker image (VM template).
-
-shallow_clone: true
-# Fetch repository as zip archive.
-
-cache:
-  - x86_64-4.8.5-release-win32-seh-rt_v4-rev0.7z
-
-environment:
-  MINGW_DIR: mingw64
-  MINGW_URL: https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/4.8.5/threads-win32/seh/x86_64-4.8.5-release-win32-seh-rt_v4-rev0.7z/download
-  MINGW_ARCHIVE: x86_64-4.8.5-release-win32-seh-rt_v4-rev0.7z
-  BUILD_TYPE: debug
-
-  matrix:
-    # TODO(someday): Add MSVC x64 builds, MinGW x86 build?
-
-    - CMAKE_GENERATOR: Visual Studio 15 2017
-      BUILD_NAME: vs2017
-      EXTRA_BUILD_FLAGS: # /maxcpucount
-      # TODO(someday): Right now /maxcpucount occasionally expresses a filesystem-related race:
-      #   capnp-capnpc++ complains that it can't create test.capnp.h.
-
-    - CMAKE_GENERATOR: Visual Studio 14 2015
-      BUILD_NAME: vs2015
-      EXTRA_BUILD_FLAGS: # /maxcpucount
-
-    - CMAKE_GENERATOR: MinGW Makefiles
-      BUILD_NAME: mingw
-      EXTRA_BUILD_FLAGS: -j2
-
-install:
-  - if not exist "%MINGW_ARCHIVE%" appveyor DownloadFile "%MINGW_URL%" -FileName "%MINGW_ARCHIVE%"
-  - 7z x -y "%MINGW_ARCHIVE%" > nul
-  - ps: Get-Command sh.exe -All | Remove-Item
-  # CMake refuses to generate MinGW Makefiles if sh.exe is in the PATH
-
-before_build:
-  - set PATH=%CD%\%MINGW_DIR%\bin;%PATH%
-  - set BUILD_DIR=build-%BUILD_NAME%
-  - set INSTALL_PREFIX=%CD%\capnproto-c++-%BUILD_NAME%
-  - cmake --version
-
-build_script:
-  - echo "Building Cap'n Proto with %CMAKE_GENERATOR%"
-  - >-
-      cmake -Hc++ -B%BUILD_DIR% -G "%CMAKE_GENERATOR%"
-      -DCMAKE_BUILD_TYPE=%BUILD_TYPE%
-      -DCMAKE_INSTALL_PREFIX=%INSTALL_PREFIX%
-  - cmake --build %BUILD_DIR% --config %BUILD_TYPE% --target install -- %EXTRA_BUILD_FLAGS%
-  # MinGW wants the build type at configure-time while MSVC wants the build type at build-time. We
-  # can satisfy both by passing the build type to both cmake invocations. We have to suffer a
-  # warning, but both generators will work.
-
-  - echo "Building Cap'n Proto samples with %CMAKE_GENERATOR%"
-  - >-
-      cmake -Hc++/samples -B%BUILD_DIR%-samples -G "%CMAKE_GENERATOR%"
-      -DCMAKE_BUILD_TYPE=%BUILD_TYPE%
-      -DCMAKE_PREFIX_PATH=%INSTALL_PREFIX%
-  - cmake --build %BUILD_DIR%-samples --config %BUILD_TYPE%
-
-test_script:
-  - timeout /t 2
-  # Sleep a little to prevent interleaving test output with build output.
-  - cd %BUILD_DIR%\src
-  - ctest -V -C %BUILD_TYPE%
+# Cap'n Proto AppVeyor configuration

```

Comment:
It appears that you've made some sort of whitespace change to every line of this file. Probably you accidentally changed the line endings to DOS-style? Can you fix your git settings so that text files are committed with standard line endings?

Reply:
Will do